src/app/hooks/useTips.ts

```diff
@@ -0,0 +1,37 @@
+import { useAccount } from "~/app/context/AccountContext";
```

Comment:
Sorry what was the reasoning behind adding this hook? To encapsulate the tips-code? Asking mainly to get a better understanding. Thought we use hooks to share code among different components but the tips aren't used somewhere else, right?

Reply:
Yes, to encapsulate the tips code but was also necessary to break when I moved `<Tips />` to its own component. The additional business logic of which tips to show (chrome browser, is Alby account, etc) is needed to be run both when looping through the tips [in the Tips component](https://github.com/getAlby/lightning-browser-extension/pull/1892/files#diff-792f8b2e81810f3e99e74179e35dcfbcd623ff5d66b44c668701e4c6f2b83781R15) but also in [`Publisher/index.tsx`](https://github.com/getAlby/lightning-browser-extension/pull/1892/files#diff-abf596d81b784f3eb9e9e7be5814d0a585cfa545c1fe3142071fdbc59e1e9e33R20) to do logic on showing messaging around the titles (if has tips show x...).  The original way had all this logic in `Publisher/index.tsx` but it was really starting to grow and seemed to make more sense to break it out.  Happy to move it back into there though if you think that makes more sense.